src/services/Auth/index.ts

```diff
@@ -16,69 +16,63 @@ const auth = new AuthRepository();
 export const createUserService = async (req: Request, res: Response) => {
     const code = generateCode();
 
-    const newUser = new User({
+    const newUser:UserType = {
         firstName: req.body.firstName,
         lastName: req.body.lastName,
         email: req.body.email,
         mobile: req.body.mobile,
         password: bcrypt.hashSync(req.body.password, 8),
         active: false,
         verificationCode: code,
-    });
-
-    const isUserExist = await User.findOne({ email: newUser.email });
+    };
+    const isUserExist = await auth.checkUserExist (req.body.email);
```

Comment:
change auth to authRepository so it depicts the name

Reply:
Changed it to authRepository class